resources/core/charts/k8s-dashboard-proxy/templates/rbac.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
```

Comment:
If you provide a SA with cluster-admin privileges here, and access to this dashboard is only limited by authentication against dex, then here is a security blocker. Any user in the kyma, with any privileges will have access to ie. all secrets in the cluster

Reply:
Yes, "cluster-admin" should be used only on minikube and "kyma-admin" on a real cluster.

Reply:
After having the discussion with @piotrmsc, we both share the same concern to not have default access for the cluster-admin privileges. 

We have decided to park this PR and have the Istio RBAC implemented later on.
When it will happen will depend upon the capacity of both teams.

Reply:
@piotrmsc How about this idea?
* Use `kyma-admin` role for the cluster deployment.
* Use `cluster-admin` role for the local/minikube deployment.

In this way, we allow the same privileges as we do with our kubeconfig.

Would it be possible for you and @radufa to discuss this on Monday and reach a conclusion?

Next steps would be to then modify this PR based on the above proposal if this is agreed upon.

Reply:
@abbi-gaurav  Have you checked wheter `kyma-admin` will be enough? The problem is not with the binding for the SA but with lack of authorization here so we have to allow this access to only certain group of people, for example with kyma-admin assigned role.

Regarding kubeconfig you download from the console, it works a bit different. Downloaded kubeconfig consists your id token and based on you're assigned RBAC role, authorization proxy checked if you have access to the requested resources so you will not be able to read secrets if your assigned RBAC role does not allow it.

Reply:
The verification if `kyma-admin` is enough still needs to be done as mentioned in the previous comment.